src/markdown-pages/docs/first-steps/initialising-home-folder.md

```diff
@@ -0,0 +1,59 @@
+
+## Initialising the Galasa home folder
+
+To initialise your Galasa home folder in your home directory along with the properties files, use the following command:
+
+```
+galasactl local init [flags]
+```
+
+This one-time command is run once per user and helps you to quickly create the folder and files that you need. If you already have a Galasa home folder set up, the folder and files are not created. 
+
+The command creates the following file structure, with default contents in each of the properties files:
+
+```
+${HOME}/.galasa
+├── bootstrap.properties
+├── cps.properties
+├── credentials.properties
+├── dss.properties
+├── overrides.properties
+```
+
+You can validate the set up by locating your user home directory and confirming that it contains a `.galasa` folder. On Windows, the user home directory resembles: ```C:\Users\<username>```, on MacOS it will be ```/Users/<username>``` and on Linux ```/home/<username>```. Note that any file or folder beginning with a `.` is a hidden folder, so you might need to change the settings on your operating system to show hidden files.
+
+
+## <a name="about"></a>About the properties files
+
+The following table explains a bit more about the purpose of the properties files that are created: 
+
+| Name |  Description  |
+| :---- | :-------- | 
+| bootstrap.properties  | The bootstrap contains the information that Galasa needs to bring up a framework in the environment to connect to the Galasa Ecosystem. If you are running a test on your local machine, the bootstrap file is blank as it does not need to refer to the ecosystem. |
+| cps.properties | The configuration property store (CPS) defines object properties, topologies, system configurations, and definitions which instruct the way in which a Galasa test runs. For example, properties for endpoints, ports and timeouts. When running in an ecosystem, all Galasa tests will use the same CPS configuration, unless any overrides are passed at submission. It is the CPS and the configurational properties that enable tests to run against multiple environments, without changing the code inside the test. | 
+| credentials.properties | It is likely that a test will need to pass credentials to the application being tested. For example, as HTTP credentials or as username and password values entered onto a 3270 screen. Rather than hard code the credentials inside a test class, you can store the values in the credentials.properties file when the test is run locally. The ability to get credentials from a credentials store means that you do not need to hard code these values inside a test, enabling the test to be run in different environments without changing a single line of code. You can extract a test property value from the CPS file for use in a test by using the `@TestProperty` annotation.|
```

Comment:
Should reference the @??? annotation which is used to get the secrets out of the credential store. Not sure how to do that yet.

Reply:
Added info about the 'getCredentials` method provided by the core manager